pylint/pyreverse/inspector.py

```diff
@@ -300,14 +300,14 @@ def visit_importfrom(self, node: nodes.ImportFrom) -> None:
             if fullname != basename:
                 self._imported_module(node, fullname, relative)
 
-    def compute_module(self, context_name: str, mod_path: str) -> int:
-        """Return true if the module should be added to dependencies."""
+    def compute_module(self, context_name: str, mod_path: str) -> bool:
+        """Should the module be added to dependencies ?"""
         package_dir = os.path.dirname(self.project.path)
         if context_name == mod_path:
-            return 0
-        if astroid.modutils.is_standard_module(mod_path, (package_dir,)):
-            return 1
-        return 0
+            return True
```

Comment:
This is a change in behaviour? `bool(0) == False`?

Reply:
Huh, nice catch. It seems no test caught that 😱 
```suggestion
            return False
```